v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -23,20 +25,31 @@ def for_reference_genome_dataset_type(
         cls,
         reference_genome: ReferenceGenome,
         dataset_type: DatasetType,
-    ) -> list['ReferenceDataset']:
+        include_queries: bool = True,
+    ) -> list[Union['ReferenceDataset', 'ReferenceDatasetQuery']]:
         reference_datasets = [
             dataset
             for dataset, config in CONFIG.items()
             if dataset_type in config.get(reference_genome, {}).get(DATASET_TYPES)
         ]
         if not Env.ACCESS_PRIVATE_REFERENCE_DATASETS:
-            return [
+            reference_datasets = [
                 dataset
                 for dataset in reference_datasets
                 if dataset.access_control == AccessControl.PUBLIC
             ]
+        if not include_queries:
```

Comment:
I think we want this to not just exclude `queries`, but also `gnomad_qc` and `gnomad_coding_and_noncoding` right?  So maybe a config flag?